src/index.js

```diff
@@ -1,32 +1,44 @@
-var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+/* var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest; // Linea comentada para correr en navegador desde index.html */
 
 var API = 'https://rickandmortyapi.com/api/character/';
```

Comment:
En el problema 2 debíamos utilizar ECMAScript 6 por lo tanto ya no utilizamos 'var'

```suggestion
const API = `https://rickandmortyapi.com/api/character/`;
```